packages/cloud-functions/src/cico/utils.ts

```diff
@@ -189,3 +189,6 @@ export const fetchLocalCurrencyAndExchangeRate = async (
 
   return result
 }
+
+export const roundDecimals = (input: number, decimals: number) =>
```

Comment:
@tarikbellamine would it make sense to use `roundDown/ roundUp` from `src/utils/formatting` instead?

Reply:
Good thought but it's in a separate package so that would be difficult to import